tests/integ/test_function.py

```diff
@@ -1176,9 +1178,16 @@ def test_date_operations_negative(session):
     assert "'DATEADD' expected Column or str, got: <class 'list'>" in str(ex_info)
 
 
+def test_get(session):
```

Comment:
we don't have test for get before?

Reply:
We only had a negative test. This is true for almost all functions in `function.py`

Reply:
ah ok, could we make it as a doctest instead of a test here?